Sage/carbon
# docs(anchor-navigation): fix stories with no example displaying in docs - FE-4387

Due to the config changes these stories are unable to access the Test stories in the production
environment.

FE-4387

### Proposed behaviour
- Examples for Anchor Navigation should load correctly.

![Screenshot 2021-10-07 at 10 48 34](https://user-images.githubusercontent.com/56251247/136361670-88de0465-665f-4e65-89de-704ae3ba965e.png)

![Screenshot 2021-10-07 at 10 48 22](https://user-images.githubusercontent.com/56251247/136361679-324a4dc2-9555-406d-8512-769fa3658aa9.png)

### Current behaviour
- Examples load with a `No Preview` error message.

![Screenshot 2021-10-07 at 10 48 48](https://user-images.githubusercontent.com/56251247/136361778-49177f09-6200-4b6d-9c95-19c007e7943b.png)

### Checklist




#### QA


### Additional context
N/A

### Testing instructions

- Ensure that the Two Anchor Navigation stories load correctly in Docs and Canvas. These are the `default` and `in full screen dialog` stories.
